### ISSUE TITLE
problems reading selected texts on apple pages on the internet 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13319]
### Steps to reproduce:
It may be that there is another way to open apple pages on windows computer, but I don't know, I will indicate the path I followed.
I installed icloud on windows computer.
I logged into my apple account.
In the start menu inside the icloud folder there is an item called pages.
I pressed enter on it, icloud opens, I navigated with the arrows until I find pages, I pressed enter.
Now with tabe I chose new, and then blank.
nvda now announces blank.
I pressed enter to enter focus mode.
Now I typed some text.
Now select a snippet of the text you just typed, for example, I pressed up and down arrows to a line with texts, pressed home to be at the beginning of the line and selected the words with ctrl + shift + right arrows. 
### Actual behavior:
nvda announces a character and does not report the selected text even after pressing nvda + shift + s 
### Expected behavior:
nvda announces selected text 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-24583,61000de2

#### Windows version:
11 22000.469
#### Name and version of other software in use when reproducing the issue:
chrome.exe, Google Chrome 97.0.4692.99

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
as an addendum nvda also announces some characters when navigating line by line with arrows, and announces blank before reading the character when walking with right and left arrows.

**Comment by Adriani90** [comment]
@fernando-jose-silva could you please test with last NVDA alpha version and Chrome 111? Are you still having this issue?

**Comment by fernando-jose-silva** [comment]
yes, everything updated and the problems continue, even if it got worse, nvda can't read consistently even when navigating through typed text with arrow keys.
It appears that the cursor does not move this is my perception which may be wrong.
To correct this, if nv access is interested, someone will have to enter pages online and see what happens, as a user with no programming knowledge, I was completely lost in this editing field.
Anyway, I got another log of the situation.
[Uploading pages.txt…]()

**Comment by fernando-jose-silva** [comment]
[pages.txt](https://github.com/nvaccess/nvda/files/11170640/pages.txt)

**Comment by Adriani90** [comment]
@fernando-jose-silva how is this now with NVDA 2025.3.1 or with the last 64bit alpha version and Chrome 142? Are you still seeing the issue? Did you already contact aple about this?

**Comment by fernando-jose-silva** [comment]
If we take the matter literally, it can be closed; selection and reading is now possible.
However, its use is problematic. When reading line by line, NVDA reads a single word on the line and then reads the rest of the text.
It's not possible to do continuous reading with NVDA + a, and when navigating character by character, NVDA pronounces two characters, I think one informed by Pages and the other by NVDA.

**Comment by Adriani90** [comment]
@fernando-jose-silva did you report this to the iCloud developers at this link?
https://www.apple.com/feedback/icloud.html

**Comment by fernando-jose-silva** [comment]
Thank you for your feedback on iCloud.
We cannot respond to you personally, but please know that your message has been received and will be reviewed by the iCloud Team. If we need to follow up with you on your ideas for improving iCloud, we will contact you directly.
We appreciate your assistance in making iCloud better.
iCloud Team
